code/modules/hallucinations/hallucinations.dm

```diff
@@ -59,3 +59,16 @@
 		else
 			for(var/mob/M in oviewers(world.view, holder))	//Only shows to others, not you; you're not aware of what you're doing. Could prompt others to ask if you're okay, and lead to confusion.
 				to_chat(M, "<B>[holder]</B> [chosen_emote]")
+
+//For adding accent tags when you imagine people talking to you
+/datum/hallucination/proc/get_hallucinated_accent(var/mob/living/talker)
+	var/hal_accent
+	if(ishuman(talker))
+		var/mob/living/carbon/human/T = talker
+		hal_accent = T.accent ? T.accent : pick(SSrecords.accents)
+	else
+		hal_accent = pick(SSrecords.accents)
+	var/datum/accent/a = SSrecords.accents[hal_accent]
+
+	var/final_icon = a.tag_icon
+	return "<IMG src='\ref['./icons/accent_tags.dmi']' class='text_tag' iconstate='[final_icon]'>"
```

Comment:
can't this just use the normal get_accent_icon proc on the mob or am i missing something?

Reply:
get_accent_icon is defined on human and I wanted to generate accents randomly for mob/living.

Reply:
Now you see, this is epic, because you can do this with the PR geeves made, but this is fine